datacommons/test/places_test.py

```diff
@@ -142,5 +206,53 @@ def test_no_dcids(self, urlopen):
     })
 
 
+class TestGetStats(unittest.TestCase):
+  """ Unit stests for get_stats. """
+
+  @mock.patch('urllib.request.urlopen', side_effect=request_mock)
+  def test_multiple_dcids(self):
+    """ Calling get_places_in with proper dcids returns valid results. """
+    # Set the API key
+    dc.set_api_key('TEST-API-KEY')
+
+    # Call get_places_in
+    stats = dc.get_stats(['geoId/05', 'geoId/06'], 'dc/0hyp6tkn18vcb')
+    self.assertDictEqual(
+        stats, {
+            'geoId/06085': ['geoId/0649670'],
+            'geoId/24031': ['geoId/2467675', 'geoId/2476650']
+        })
+
+  @mock.patch('urllib.request.urlopen', side_effect=request_mock)
+  def test_bad_dcids(self):
+    """ Calling get_places_in with dcids that do not exist returns empty
+      results.
+    """
+    # Set the API key
+    dc.set_api_key('TEST-API-KEY')
+
+    # Call get_places_in with one dcid that does not exist
+    bad_dcids_1 = dc.get_stats(['geoId/05', 'dc/MadDcid'], 'dc/0hyp6tkn18vcb')
+    self.assertDictEqual(bad_dcids_1, {
+        'geoId/06085': ['geoId/0649670'],
+        'dc/MadDcid': []
+    })
```

Comment:
I'm puzzled by the checks. Why don't the expected dict contain time-series?

Also s/get_places_in/get_stats/

Reply:
Sorry, I prematurely requested your review. Was having trouble testing locally so Bo had me create a PR for test to run on CloudRun, shouldn't have requested, though. 

This should be fixed now, but need to see test results.